### PR TITLE
Update Black checking process and fix output in PR

### PR DESCRIPTION
**Pull request by naveen521kk**
## List of Changes
- Make Black check only modified file
- Fix errors in a PR comments.

## Motivation
- Tests are failing and should be passed.

## Testing Status
Tested in my fork https://github.com/naveen521kk/manim/pull/23 and https://github.com/naveen521kk/manim/pull/24

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

### CODE REVIEW

**Comment by leotrs** [comment]
:tada: :rocket: :tada: :rocket: :tada: :rocket: :tada: :rocket: :tada: :rocket: :tada: :rocket:

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -55,25 +52,24 @@ jobs:
...
-          body="This an automated pull requests. Files Changed are `n `n ```````n$($env:file_change)`n``````"
+          body="This an automated pull requests. Files Changed are `n `n ```````n$($filechange)`n``````"
           } | ConvertTo-Json -Compress
           echo "$postContents"
           Invoke-WebRequest `
             -Method POST `
             -Headers @{ "Accept" = "application/vnd.github.v3+json"; "Authorization"= "token $($env:access_token)"}`
-            -Uri https://api.github.com/repos/ManimCommunity/manim/pulls `
-            -Body "$($postContents)"
+            -Uri https://api.github.com/repos/naveen521kk/manim/pulls `
```
Comment (leotrs) [inline .github/workflows/black.yml:74]: Question: this looks like a typo, no? Must it always run on your fork?

**Comment by leotrs** [comment]
Also, it's not a huge deal, but CodeFactor is asking you to add newlines at the end of some files....

File: .github/workflows/black.yml (hunk 1 above)
Comment (naveen521kk) [reply .github/workflows/black.yml:74]: Ok this is a typo. Sorry.

**Review by leotrs — APPROVED** [review]

**Review by kilacoda — APPROVED** [review]

File: .github/workflows/black.yml [hunk 2]
```diff
@@ -55,25 +52,24 @@ jobs:
...
           $blackVersion = python -c "print('$($blackVer)'.split()[2])"
           $date = python -c "import time;print(time.localtime().tm_mday)"
           $pushBranch="black-$($blackVersion)-$($date)"
+          $filechange=(gc "D:/diffmod.txt").split() -join "`n`n"
           $postContents = @{title="Automated PR: Autoformat with black";
-          head="ManimCommunity:$($pushBranch)";
+          head="manimcommunity:$($pushBranch)";
           base="master";
-          body="This an automated pull requests. Files Changed are `n `n ```````n$($env:file_change)`n``````"
+          body="This an automated pull requests. Files Changed are `n `n ```````n$($filechange)`n``````"
```
Comment (Aathish04) [inline .github/workflows/black.yml:68]:
```suggestion
          body="This an automated pull request. Files Changed are `n `n ```````n$($filechange)`n``````"
```

**Review by Aathish04 — APPROVED** [review]

**Pull request merged**